vnext/src/IntegrationTests/AsyncStorageTest.windesktop.js

```diff
@@ -43,15 +44,32 @@ function expectTrue(condition: boolean, message: string) {
   }
 }
 
+// Type-safe wrapper around JSON.stringify
+function stringify(
```

Comment:
>stringify [](start = 9, length = 9)

Should we name the function to say that it is JSON related?

Reply:
No.

This file is forked from RN's original.
We should keep this as non-diverging as possible.

Reply:
I see. Good to know.